indexing-service/src/test/java/org/apache/druid/indexing/overlord/TaskLockboxTest.java

```diff
@@ -1447,51 +1423,43 @@ public void testAppendLockCompatibility() throws Exception
         MEDIUM_PRIORITY
     );
 
-    // An exclusive lock cannot be created for an overlapping interval
     validator.expectLockNotGranted(
         TaskLockType.EXCLUSIVE,
         Intervals.of("2017-05-01/2017-06-01"),
         MEDIUM_PRIORITY
     );
 
-    // A shared lock cannot be created for an overlapping interval
     validator.expectLockNotGranted(
         TaskLockType.SHARED,
         Intervals.of("2016/2019"),
         MEDIUM_PRIORITY
     );
 
-    // A replace lock cannot be created for a non-enclosing interval
     validator.expectLockNotGranted(
         TaskLockType.REPLACE,
         Intervals.of("2017-05-01/2018-01-01"),
         MEDIUM_PRIORITY
     );
 
-    // A replace lock can be created for an enclosing interval
     final TaskLock replaceLock = validator.expectLockCreated(
         TaskLockType.REPLACE,
         Intervals.of("2017/2018"),
         MEDIUM_PRIORITY
     );
 
-    // Another replace lock cannot be created
     validator.expectLockNotGranted(
         TaskLockType.REPLACE,
         Intervals.of("2016/2019"),
         MEDIUM_PRIORITY
     );
 
 
-    // Any append lock can be created, provided that it lies within the interval of the previously created replace lock
-    // This should not revoke any of the existing locks even with a higher priority
```

Comment:
This comment was actually useful fwiw...  It's explaining an intention that is not immediately apparent from the code.  It was a good example of when to write a comment.